### PR TITLE
fix typo in docs/topics/spiders

### PR DESCRIPTION
**Pull request by KristobalJunta**

### CODE REVIEW

File: docs/topics/spiders.rst [hunk 1]
```diff
@@ -692,7 +692,7 @@ SitemapSpider
 
     .. method:: sitemap_filter(entries)
 
-        This is a filter funtion that could be overridden to select sitemap entries
+        This is a filter function that could be overridden to select sitemap entries
```
Comment (elacuesta) [inline docs/topics/spiders.rst]: May I suggest an additional change?

```suggestion
        This is a filtering function that could be overridden to select sitemap entries
```

Comment (kmike) [reply docs/topics/spiders.rst]: Both ways of writing it look fine to me.

**Comment by kmike** [comment]
Merging it, as it is a clear improvement as-is.

**Comment by kmike** [comment]
Thanks @KristobalJunta!

**Comment by KristobalJunta** [comment]
@kmike you're welcome! Since I started reading scrapy code often recently, I suppose I'll be back with more PR's :)

**Pull request merged**